src/sys/mman.rs

```diff
@@ -351,6 +363,29 @@ pub unsafe fn mremap(
     }
 }
 
+#[cfg(target_os = "netbsd")]
```

Comment:
This function already exists on Linux, and it looks to me like the only different is the operand order.  In the interest of portability, could you please share a single signature for the two operating systems?  You can use `cfg_if!` around the libc syscall to handle the different operand orders.